app/webpacker/components/Results/Rankings/RankingsTable.jsx

```diff
@@ -57,69 +57,80 @@ function ResultRow({
 export default function RankingsTable({
   rows, competitionsById, isAverage, show,
 }) {
-  const r = useMemo(() => {
+  const results = useMemo(() => {
     let rowsToMap = rows;
     let firstContinentIndex = 0;
     let firstCountryIndex = 0;
     if (show === 'by region') {
       [rowsToMap, firstContinentIndex, firstCountryIndex] = rows;
     }
```

Comment:
I would do something like this to remove these `let`s too, but it's somewhat personal preference.
```suggestion
    const isByRegion = show === 'by region';
    const rowsToMap = isByRegion ? rows[0] : rows;
    const firstContinentIndex = isByRegion ? rows[1] : 0;
    const firstCountryIndex = isByRegion ? rows[2] : 0;
```
If the value can be calculated directly, then it's usually clearer, and removes the possibility of a future PR updating the `let` in an obscure/unexpected place. And if you can't directly calculate the value, then it's usually because you're using a loop instead of a reducer/map, as was the case below.

Reply:
Or if you prefer,
```suggestion
    const isByRegion = show === 'by region';
    const [rowsToMap, firstContinentIndex, firstCountryIndex] = isByRegion ? rows : [rows, 0, 0];
```

Reply:
For clarity, it might help to split the region case out into a separate case/helper function. Since the rank stuff is only used for persons/results, and the country/continent stuff is only used for by region. Not sure if it's worth it at this point though.